canvas_modules/common-canvas/src/common-properties/components/truncated-content-tooltip/truncated-content-tooltip.jsx

```diff
@@ -22,8 +22,27 @@ import { has } from "lodash";
 
 // Reusable component to show tooltip if the content is truncated
 export default class TruncatedContentTooltip extends React.Component {
+	// Return true if the string can be displayed in the available space
+	// Return false if the string is truncated and ellipsis is shown on the UI
+	// (offsetWidth) is a measurement in pixels of the element's CSS width, including any borders, padding, and vertical scrollbars
+	// (scrollWidth) value is equal to the minimum width the element would require
+	//  in order to fit all the content in the viewport without using a horizontal scrollbar
+	canDisplayFullText(elem) {
+		if (elem) {
```

Comment:
I would expect this to be `elem` or `truncatElem`.  Then textfield passes in `truncatElem`